.circleci/config.yml

```diff
@@ -7,7 +7,10 @@ jobs:
   release-patch-with-changelog:
     machine:
       image: ubuntu-1604:201903-01
-      docker_layer_caching: true
+      # This job has been blocked because Docker Layer Caching is not available on your plan.
+      # Please upgrade to continue building.
+      # Note: you will need to push a new commit or call the API to rerun the pipeline.
+      #docker_layer_caching: true
```

Comment:
@exequielrafaela Instead of commenting out this line you could set it to false. Minor suggestion though.

Reply:
@diego-ojeda-binbash OJ you're totally right, I was in a bit of a hurry and decided to commented as first choice. I'll updated here and considered it for all the other CircleCi jobs which currently have this line commented. 

Thanks for taking the time here! :handshake:

Reply:
:+1: 

```yaml
jobs:
  #
  # Release
  #
  release-patch-with-changelog:
    machine:
      image: ubuntu-1604:201903-01
      # This job has been blocked because Docker Layer Caching is not available on your plan.
      # Please upgrade to continue building.
      # Note: you will need to push a new commit or call the API to rerun the pipeline.
      docker_layer_caching: false
```